modules/controlfreec/1.1/controlfreec.smk

```diff
@@ -33,13 +37,17 @@ localrules:
     _controlfreec_all
 
 
+wildcard_constraints:
```

Comment:
Is there a particular rule/rules that need this wildcard constraint? I've had TONS of issues setting global constraints in my Snakefiles and I've reverted to using them only on a per-rule basis because of it. If you can set this just in one or two rules that need it it might save a ton of headaches down the road.

Reply:
I based this off your gridss snakemake where you had it as a global constraint. I didn't have any issues with this wildcard since my test set only had unmatched and matched. I'll remove it for now since it doesn't seem like it has any impact at all.

Reply:
Yes, I've only recently learned of the problems that these global constraints cause. Thanks for removing.